### DISCUSSION TITLE
🎉 写在 2025 跨年之际

### DISCUSSION THREAD

**Discussion by sentsim** [layui/layui#2448] [分享]
2024 即将成为往昔，而我们持续了八年之久的全浏览器兼容的 Layui 2 也终于即将告一段落。

趁此 2025 跨年之夜，特别感谢 @Sight-wcg 在过去一年为 Layui 所作出的巨大努力，他的出现宛如一颗启明星，打破了 Layui 的漫漫长夜，为社区注入了前所未有的活力，巧合的是，Sight 的英文昵称正是 Morning Star，所以我习惯称他为晨星，意指启明之光。

晨星大概从 `2.9.0` 开始，几乎以一己之力，贡献了超过 95% 以上的更新，其中改善的细节之众，提升的逻辑之巧，让年迈的 Layui 2 重焕生机，逐渐也具备了应对更多复杂 Web 应用的能力。

不仅如此，在晨星的提议下，Layui 的维护工作从一年前便已由 Gitee 全线迁移到 Github，我们创建了贡献者指南 和 CI 流程，用于保证 Issue 和 PR 的高效。得益于各项工作的有序展开，Layui 的版本迭代也变得有规律有节奏。

接下来，Layui 2 将作为 stable 版本长期存在，后续除重要 BUG 外，基本不再对其进行大幅修改。而我们的主要精力也将放在构建全新版本上，继续维持 Layui 存在的合理性，维持一种温度、一种希望。

—— 贤心 2024/12/31

**Comment by sggcn** [comment]
👍👍👍👍👍加油

**Comment by DrLiu123** [comment]
wow  LayUI 3.0要来了吗！

**Reply by ByYogi** [reply]
📂新建文件夹
> 为避免给开发团队带来压力，还请大家不要过多地催问进度。3.0 的开发周期可持续关注： [Github Milestones](https://github.com/layui/layui/milestone/1).

别催，快了

**Comment by swanfly** [comment]
加油💪🏻
